pyOpenSci/update-web-metadata
# Move Token Import From a Pickle File Import to a Simple Text File

# What was done
---

Addressing issue #21

- Updated the code in _parse_review_issues.py_ to import a txt file and populate the API_TOKEN variable rather than using a pickle.
- Created a small helper function in the _pyosmet src/file_io.py_ to import the parse the text file.
- Updated the code in all three scripts as well to use the txt file rather than the pickle file
-- _parse_review_issues.py_
-- _parse-contributors.py_
-- _update_reviewers.py_
- Updated the _.gitignore_ file to ensure the txt file is never populated into git history
- Updated _README.md_ with instructions in creating the token and storing it for use.

# How to test
---

- Run the workflow (particularly parse_review_issues.py) and review the output file (package.yml).
- Ensure that no error is encountered related to the token or token file.

...

Special thanks to the team at PyCon 2023 for making this possible, with their help and encouragement! Thank you especially to @lwasser.
